server/channels/api4/channel.go

```diff
@@ -724,6 +726,10 @@ func getChannelsMemberCount(c *Context, w http.ResponseWriter, r *http.Request)
 		c.Err = model.NewAppError("getChannelsMemberCount", model.PayloadParseError, nil, "", http.StatusBadRequest).Wrap(sortErr)
 		return
 	}
+	if len(channelIDs) > maxListSize {
+		c.SetInvalidParam("channel_ids")
+		return
+	}
```

Comment:
Based on the previous, I thiught this limit was removed.. I'm fine with keeping it, we just need to document it